app/domain/errors.rb

```diff
@@ -496,6 +496,11 @@ module ResourceRestrictions
         code: "CONJ00049E"
       )
 
+      EmptyAnnotationGiven = ::Util::TrackableErrorClass.new(
+        msg: "Empty value for '{0-annotation-name}' has been given",
```

Comment:
Hey @shulifink @hilagross 
Can i get review on this error message?

Reply:
I would write: Value '{0-annotation-name}' is empty
but would like to hear @shulifink